public/app/core/components/Page/Page.tsx

```diff
@@ -47,7 +47,7 @@ class Page extends Component<Props> {
     const { navModel, children, ...otherProps } = this.props;
     return (
       <div {...otherProps} className="page-scrollbar-wrapper">
-        <CustomScrollbar autoHeightMin={'100%'} className="custom-scrollbar--page">
+        <CustomScrollbar autoHeightMin={'100%'} page={true}>
```

Comment:
Good catch, but do you think page a suitable name for this property? I think it seems a bit too generic?